.travis.yml

```diff
@@ -0,0 +1,13 @@
+git:
+  depth: 10
+branches:
+  only:
+    - master
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+os: osx
+
+script: apm install && npm test
```

Comment:
This is failing for a few different reasons, the primary being you never installed Atom :stuck_out_tongue:.